libs/core/src/components/pds-select/stories/pds-select.docs.mdx

```diff
@@ -0,0 +1,127 @@
+import { Meta, Canvas, ArgTypes } from '@storybook/blocks';
+import { html, render } from 'lit-html';
+
+import * as stories from './pds-select.stories.js';
+
+<Meta of={stories} />
+
+# Select
+
+Select allows users to make a single or multiple selections from a list of options.
+
+## Sub Components
+
+### Select Option
+
+A subcomponent of the select component. This component is used to define the options of the select component.
+
+Requires a unique `component-id` property to be passed.
+
+e.g. `<pds-select-option component-id="myId" value='My value'></pds-select-option>`
+<Canvas mdxSource={`
+<pds-select-option component-id="myId" value='My value'></pds-select-option>
+`}>
+  <pds-select-option component-id="myId" value='My value'></pds-select-option>
+</Canvas>
+
+#### Value vs Text Content
+Requires either a `value` property or text content to be passed. If both are passed, the `value` property will be used.
+
+e.g. `<pds-select-option component-id='myId' value='my value'>My text content</pds-select-option>`
+
+<Canvas mdxSource={`
+<pds-select-option component-id='myId' value='my value'>My text content</pds-select-option>
+`}>
+  <pds-select-option component-id='myId' value='my value'>My text content</pds-select-option>
+</Canvas>
+
+will render `my value` as the option value.
+
+## Guidelines
+### Accessibility
+Our select component doesn't use the native HTML `<select>`, instead ARIA is used to achieve semantic HTML behavior.
+This select is keyboard accessible following the Comboxbox and Listbox best practices from the W3C. This allows the select component to also be screen reader accessible.
+
+- Specify a unique `component-id` for  the `pds-select`. This will properly set the relationship between the menu button and listbox.
+- Specify a unique `component-id` for each `pds-select-option` to achieve form selection through ARIA roles.
+
+## Properties
+
+<ArgTypes of={stories} />
+
+### Default
```

Comment:
I called it out in the code, but should definitely call it out in text since my stuff is behind a click
![Screen Shot 2023-09-11 at 4 59 55 PM](https://github.com/Kajabi/pine/assets/1241836/1fb57654-1945-4712-8628-2314c6f72902)